### ISSUE TITLE
Restarting my pc delete hotkeys

### ISSUE THREAD

**Issue by santimonti** [xenolightning/AudioSwitcher_v1#2213]
Hello, im new to Audioswitcher but I'm having a problem with it, every time I restart my pc one of the hotkets disappear and the worst part is that, if i want to create again the same hotkey, I get a warn message saying that is a duplicate hotkey.
Is there any solution?

- [ 1.8.0.142] I have listed the Audio Switcher version
- [W10 2004 ] I have listed my operating system, and version

**Comment by Razor5284** [comment]
Me too.

**Comment by NeTgHoStKiLlEr** [comment]
Me three! BTW @santimonti you can click the clear all button before trying to redo your hotkeys and it should work then. But a fix would be nice :) Today, after a BSOD (after stupidly changing V+Core in Dragon Center.. ), the software failed to automatically start up. Looks like not just the hotkeys were gone, but all the other settings as well?

**Comment by Eduzyrtin** [comment]
I am having this problem too

**Comment by ghost** [comment]
I actually add to this I wanted to open a new ticket about it.
It's not just this but the hotkeys can get corrupted (eg your key does NOT work anymore even disappears from the list but when you try to readd the same key it wont work because its in the config file so you need to delete the whole config file out).

Can't you fix such a simple functionality?

I mean this program don't have to do hundred quadtrillion things like systemd just one thing and you can't even get this right ::MAD::

**Comment by m-aps** [comment]
Same issue here.
feraldruid22: you can click Clear All to free the hotkey from the config file, then you can use it again

**Comment by vallcrist** [comment]
Yep, I've had this issue. I submitted a fix that is live already, I believe.

Enable the checkbox for "Show Unknown Devices in hotkey list" and you should be able to re-setup the hotkeys that vanished

https://github.com/xenolightning/AudioSwitcher_v1/pull/2169
